utils/addKeyboardListrener/addKeyboardListrener.ts

```diff
@@ -0,0 +1,28 @@
+export const addKeyboardListener = (
+  combinations: { key: string; ctrl: boolean | null },
+  callback: (props: object) => void,
+): { clear: () => void } => {
+  window.addEventListener('keydown', (event) => {
+    if (combinations.ctrl === null) {
+      if (event.key === combinations.key) {
+        callback(event);
+      }
+    } else if (event.key === combinations.key && event.ctrlKey === combinations.ctrl) {
+      callback(event);
+    }
+  });
+  const clear = (): void => {
```

Comment:
this will not clear the listener, you need callback link which was added, you are creating a new function and links are different for functions